source/includes/_01-your-first-backend-in-10-minutes.md

```diff
@@ -0,0 +1,204 @@
+
+## 7. Testing
+Let's get started testing the project. We will perform two actions:
+- Add a couple of posts
+- Retrieve all posts
+- Retrieve a specific post
+
+To perform the HTTP requests, you might want to use something like `Postman`, although `curl` would also work.
+
+### 7.1 Creating posts
+We will perform a two `POST` request to the following commands URL:
+
+`https://<api-gateway-id>.execute-api.us-east-1.amazonaws.com/production/commands`
+
+The first request:
+
+```json
+{
+	"typeName": "CreatePost",
+	"version": 1,
+	"value": {
+    	"postId": "95ddb544-4a60-439f-a0e4-c57e806f2f6e",
+    	"title": "This is my first post",
+    	"content": "I am so excited to write my first post",
+		"author": "Some developer"
+  }
+}
+```
+
+The second request:
+
+```json
+{
+	"typeName": "CreatePost",
+	"version": 1,
+	"value": {
+    	"postId": "05670e55-fd31-490e-b585-3a0096db0412",
+    	"title": "This is my second post",
+    	"content": "I am so excited to write my second post",
+		"author": "Some developer"
+  }
+}
+```
+
+We should now have two `Posts` in our backend, no authorization header is required since we have allowed `all` access to our commands and read models.
+
+### 7.2 Retrieving all posts
+In order to retrieve the information we just sent, lets perform a `GET` request that will be hitting our read model `PostReadModel`:
+
+`https://<api-gateway-id>.execute-api.us-east-1.amazonaws.com/production/readmodels/PostReadModel`
+
+### 7.3 Retrieving specific post
+It is also possible to retrieve specific a `Post` by adding the `postId` to the URL, e.g.:
+
+`https://<api-gateway-id>.execute-api.us-east-1.amazonaws.com/production/readmodels/PostReadModel/95ddb544-4a60-439f-a0e4-c57e806f2f6e`
+
+## 8. Removing stack
+If you want to un-deploy your backend, just execute the following command:
+
+`boost nuke -e production`
+
+It will ask you to verify the project name, it will be the same one that it was written when we created the project. If you don't remember the name, go to `<project-root>/package.json` and copy the `name`. 
+
+## 9. Further improvements
+The are many other options for your serverless backend built with Booster Framework:
+- GraphQL API available with subscriptions
+- Securing requests depending on user roles
+- Events that trigger more events
+- Reading entity snapshots in handlers to apply domain-driven decisions
+- and much more...
+
+## 10. Conclusion
+If you reached this point following all the steps you have created a serverless backend in less than 10 minutes. This framework intends to simplify the tedious process of building a backend from scratch. Furthermore, it reduces the infrastructure management work by automatically provisioning anything your project needs. It is not just magic, it is Booster Framework.
+
+We hope you have enjoyed it and we would like to remind you that Booster Framework is an open-source project and your contributions to make it better will be much appreciated.
+
+Thank you very :D
```

Comment:
I love this section, although I'm not sure if the tone fits under the documentation, given that the user will keep scrolling down. What do you think? Should we rephrase it? Should we remove it?

Reply:
I think we should rephrase it to something like, "Congratulations! you've built a serverless backend in less than 10 minutes. We hope you have enjoyed discovering the magic of Booster Framework, please keep reading if you want to know more about it."

Reply:
I removed the conclussion section and added a green `aside`